android/build.gradle

```diff
@@ -21,6 +21,7 @@ android {
 
 dependencies {
   compile fileTree(include: ['*.jar'], dir: 'libs')
+  compile 'com.android.support.constraint:constraint-layout:1.0.2'
```

Comment:
Seems you don't need CL here. it can be accomplished by using LL. Correct me if i'm wrong. Minor.